Sources/Endpoints/TestFlight/Apps/ListApps.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  ListApps.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Pascal Edmond on 16/11/2018.
+//
+
+import Alamofire
+
+extension APIEndpoint {
+    
+    /// Find and list apps added in App Store Connect.
+    ///
+    /// - Parameters:
+    ///   - fields: Fields to return for included related types.
+    ///   - filters: Attributes, relationships, and IDs by which to filter.
+    ///   - relationships: Relationship data to include in the response.
+    ///   - sortBy: Attributes by which to sort.
+    ///   - limits: Number of included related resources to return.
+    public static func apps(
+        select fields: [ListApps.Field]? = nil,
+        filters: [ListApps.Filter]? = nil,
+        include relationships: [ListApps.Relationship]? = nil,
+        sortBy: [Sorting<ListApps.SortableField>]? = nil,
+        limits: [ListApps.Limit]? = nil) -> APIEndpoint<AppsResponse> {
+        var parameters = [String: Any]()
+        if let fields = fields { parameters.merge(with: fields) }
+        if let filters = filters { parameters.merge(with: filters) }
+        if let relationships = relationships { parameters.merge(with: relationships) }
+        if let sortBy = sortBy { parameters.merge(with: sortBy) }
+        if let limits = limits { parameters.merge(with: limits) }
+        return APIEndpoint<AppsResponse>(path: "apps", method: .get, parameters: parameters)
```

Comment:
Same here, can we write tests for this to verify the outcoming request?

Reply:
Let me know if it's missing something : [ListAppsTests.swift](https://github.com/AvdLee/appstoreconnect-swift-sdk/blob/dabddc3c7833ab8b4c7f64ce54982747438c926b/Example/CocoaPods-AppStoreConnect-Swift-SDK/Tests/ListAppsTests.swift)